### PR TITLE
Adding start-hapiness as a Boilerplate

### PR DESCRIPTION
**Pull request by thebergamo**
Fix boilerplates sort and adding start-hapiness

### CODE REVIEW

File: lib/resources.js [hunk 1]
```diff
@@ -60,9 +64,10 @@ exports.categories = {
...
             description: 'Full-stack boilerplate with Node.js, hapi, PostgreSQL and AngularJS (MEAN.js port)'
         },
-        'hapi-api': {
-            url: 'https://github.com/rjmreis/hapi-api',
-            description: 'Lean hapi API Boilerplate with an opinionated view on project structure'
+        'start-hapiness' : {
+            url: 'http://github.com/thebergamo/start-hapiness',
+            description: 'A Boilerplate that help you to create fast project with Hapi + MongoDB'
+        }
         }
```
Comment (nlf) [inline lib/resources.js]: you have a syntax error here with an extra `}`

Comment (thebergamo) [inline lib/resources.js]: Sorry for that :(

**Comment by thebergamo** [comment]
@nlf fixed

**Comment by nlf** [comment]
thanks!

**Pull request merged**